core/dbt/cli/main.py

```diff
@@ -73,6 +75,13 @@ def cli(ctx, **kwargs):
     if flags.RECORD_TIMING_INFO:
         ctx.with_resource(profiler(enable=True, outfile=flags.RECORD_TIMING_INFO))
 
+    cli_vars = parse_cli_vars(getattr(flags, "vars", "{}"))
```

Comment:
Will `flags.vars` ever be a valid attribute reference? My understanding is that [Flags sets attribute names as uppercase](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/flags.py#L29).

Any reason not to default the [`vars` param to `{}`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/params.py#L347-L352)? That way this line could be simplified to: 
```python
cli_vars = parse_cli_vars(flags.VARS)
```